packages/pyolite-kernel/src/worker.ts

```diff
@@ -309,6 +342,10 @@ self.onmessage = async (event: MessageEvent): Promise<void> => {
       results = await execute(messageContent);
       break;
 
+    case 'input-request':
```

Comment:
This should be removed.

Reply:
And replaced by `case 'input-reply'`, in this case resolve the Promise that your `input` function created so that it can continue